src/conn/conn_sweep.c

```diff
@@ -266,7 +270,8 @@ __sweep_server(void *arg)
 		WT_ERR(__sweep_mark(session, &dead_handles));
 
 		if (dead_handles == 0 &&
-		    conn->open_file_count < conn->sweep_handles_min)
+		    (conn->open_file_count < conn->sweep_handles_min ||
```

Comment:
Add a comment explaining the conditions in this statement. It's subtle enough that I had to think quite hard about it.